volume_scroller@francislavoie.github.io/extension.js

```diff
@@ -62,13 +68,17 @@ export default class VolumeScrollerExtension extends Extension {
   _handle_scroll(_actor, event) {
     let volume = this.sink.volume;
 
-    switch (event.get_scroll_direction()) {
+    const inverted = this._get_direction();
+    const multiplier = inverted ? -1 : 1;
+
+    const scrollDirection = event.get_scroll_direction();
+    switch (scrollDirection) {
```

Comment:
Probably don't need the extra method, can simplify like this

```suggestion
    const multiplier = this.direction ? -1 : 1;
    switch (event.get_scroll_direction()) {
```